samples/sensor/veaa_x_3/boards/nucleo_h563zi.overlay

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024, Vitrolife A/S
+ */
+
+/* spi1 sck conflicts with dac1 channel 3 */
+/delete-node/ &spi1;
+
+/ {
+	test_veaa_x_3: test_veaa_x_3 {
+		status = "okay";
+		compatible = "festo,veaa-x-3";
+		io-channels = <&adc1 3>;
+		dac = <&dac1>;
+		dac-channel-id = <2>;
```

Comment:
can't the DAC go in the io-channels property as well? and use io-channels-names property?

Reply:
Not sure. Have not thought about that. Do you mean `io-channels = <&adc1 3 &dac1 2 12>;`?

Reply:
The DAC controller binding does describe io-channel cells and the stm32 dac binding has "output" as a cell, but I guess now I see the DAC header does not actually have a macro to easily get a DAC spec by the DT like for example the adc.h header has.... so I will say what I recommended actually outside the scope of the PR as I didn't realize the DAC subsystem did not have this support fully implemented yet, so nevermind if you don't want to tackle it here